opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -251,6 +257,10 @@ impl<RT: Runtime> PeriodicReaderWorker<RT> {
     async fn process_message(&mut self, message: Message) -> bool {
         match message {
             Message::Export => {
+                otel_debug!(
+                    name: "PeriodicReader.ExportMessageReceived",
```

Comment:
`Message` and channels are implementation related terms. I think we should use more generic and intuitive terms for event names such as these:
- `PeriodicReader.ExportTriggered`
- `PeriodicReader.ForceFlushCalled`
- `PeriodicReader.ShutdownCalled`

Reply:
Not related to this comment, but just wondering if it should be fine to include implementation-specific terms in `debug` macros, as long as they're not intended for the end user?

Reply:
I think its inevitable that we need to use implementation specific terms in some places. My only goal is to restrict them to TRACE/DEBUG level logs only.

Reply:
Let's use user-friendly names wherever possible for any log level? It would make the collected logs more readable. It would also save the effort of renaming event names, if at all we change the underlying implementation details. 

We can use implementation details in the event name, when it's indeed inevitable.

Reply:
Yes, that's the plan.

Reply:
This PR is just adding some debug logs, so I can see some logs without having to "trigger" an error condition to see a log, and see if self-diag example can be removed. (https://github.com/open-telemetry/opentelemetry-rust/pull/2274#issuecomment-2456179200)